src/struct/HTTP.ts

```diff
@@ -21,6 +21,128 @@ export class HTTP {
 			});
 		});
 
+		this.app.get("/imageApi", async (req, res) => {
+			const { xp, xpToLevel, level, avatarURL, tag, status } = req.query;
+			const { createCanvas, loadImage, registerFont } = require("canvas");
+			const post = {
+				nickname: tag,
+				rank: level,
+				xp: xp,
+				toLevel: xpToLevel,
+				avatar: avatarURL
+			};
+
+			const width = 840;
+			const height = 214;
+
+			const canvas = createCanvas(width, height);
+			const context = canvas.getContext("2d");
+			registerFont(__dirname + '/../assets/fonts/Poppins-Bold.ttf', { family: 'Poppins Bold' });
```

Comment:
As far as I can see from here, we are just using 3 ttf files but you uploaded 18 different ttf files. Can you delete the ones that we don't use?